mfrlibs_rpi.c

```diff
@@ -40,7 +40,9 @@ const char defaultProductClass[] = "RDK";
 const char defaultSerialNumber[] = "0000000000000000";
 const char defaultHardwareVersion[] = "RASPBERRYPI_3_V01";
 const char defaultSoftwareVersion[] = "2.0";
-const char defaultMacAddress[] = "000000000000";
+const char defaultMacAddress[] = "00:00:00:00:00:00";
+const char defaultBluetoothMacAddress[] = "00:00:00:00:00:00";
```

Comment:
Please use a common const variable for all default values to reduce the memory requirement.